ansible/roles/kong-api/defaults/main.yml

```diff
@@ -4834,6 +4871,24 @@ kong_apis:
     - name: request-size-limiting
       config.allowed_payload_size: "{{ medium_request_size_limit }}"
 
+  - name: collectionPrivateRead
+    uris: "{{ collection_prefix }}/v1/private/read"
+    upstream_url: "{{ content_service_url }}/collection/v4/private/read"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'anonymousContentAccess'
```

Comment:
Is this API accessed by users / applications without the user logging in? What private data would an anonymous user read?